packages/cli/src/tools/releaseChecker/releaseCacheManager.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @flow
+ */
+import path from 'path';
+import fs from 'fs';
+import logger from '../logger';
+
+type ReleaseCacheKey = 'eTag' | 'lastChecked' | 'latestVersion';
+
+function loadCache(): ?Object {
+  try {
+    const cacheRaw = fs.readFileSync(path.resolve(__dirname, '.cache'), 'utf8');
```

Comment:
We can't store cache like this in user directory because they'll need to git-ignore it. Please use `os.tmpdir()`

Reply:
But shouldn't `path.resolve(__dirname, '.cache')` always resolve to `MyProject/node_modules/.../cli/build/tools/releaseChecker/.cache`? If so, the cache should already be ignored.

Reply:
You're right, I somehow missed the `__dirname` there

Reply:
Quick question about the cache folder: I am not sure if I remember it correctly but when running `yarn install` (or yarn add / remove), yarn replaces the contents of the dependencies inside node_modules with its cached version. Does this mean the `.cache` is always cleared after an yarn install?

Reply:
Good question! I'm unaware of the behavior of yarn when it comes to already installed packages, but if you are correct then yes, the cache would be cleared. We should investigate how both Yarn and Npm actually behave.

Reply:
We could use `os.tmpdir()` to completely eliminate any doubts about this but we would need to create a cache file per project (maybe inferring the file name from the project name?).

Reply:
A quick test showed me that yarn replaces the node_modules when the node_modules is not up to date with the yarn.lock / package.json, this applies to yarn install / add / remove. 

I am not sure if this behaviour is officially documented. At least the `patch-package` mentioned it: https://www.npmjs.com/package/patch-package#why-use-postinstall-postinstall-with-yarn

I am not sure about npm.

Reply:
About `os.tmpdir()` on macOS, I think this folder gets cleared at least every reboot on macOs. You can compare the content of `ls $TMPDIR` with the content it after reboot.

Why not create a `.react-native-cli` folder in `os.homedir()` to store the .cache dir in? This way you have more control over how long you want to retain the cache